runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1554,7 +1554,17 @@ gcParseCommandLineAndInitializeWithValues(J9JavaVM *vm, IDATA *memoryParameters)
 			}
 		}
 		/* set default max heap for Java */
-		extensions->computeDefaultMaxHeapForJava(enableOriginalJDK8HeapSizeCompatibilityOption);
+		extensions->memoryMax = extensions->computeDefaultMaxHeapForJava(enableOriginalJDK8HeapSizeCompatibilityOption);
+		extensions->maxSizeDefaultMemorySpace = extensions->memoryMax;
+	}
+	result = option_set_to_opt(vm, OPT_XMX, &index, EXACT_MEMORY_MATCH, &extensions->userSpecifiedParameters._Xmx._valueSpecified);
```

Comment:
This option is parsed at line 1605. Why you have added extra parsing here ?!

Reply:
I just want to follow the order of header and add an option inside the userSpecifiedParameters, will remove the previous one if no problem